### PR TITLE
replace 'keep_dims' with 'keepdims' in quantiles.py

### LINKED ISSUE #557: tfp.stats.percentile uses keep_dims instead of keepdims
I think the rest of TF has converged on `keepdims` as the keyword argument name instead of `keep_dims`. It would be nice to switch over.

### PR DESCRIPTION
**Pull request by rushabh-v**
resolves #557

### CODE REVIEW

**Comment by rushabh-v** [comment]
`continuous-integration/travis-ci/pr` files with `connection reset by peer` 
what should I do in this case?

File: tensorflow_probability/python/stats/quantiles.py [hunk 1]
```diff
@@ -406,7 +406,7 @@ def percentile(x,
                q,
                axis=None,
                interpolation=None,
-               keep_dims=False,
+               keepdims=False,
```
Comment (brianwa84) [inline tensorflow_probability/python/stats/quantiles.py]: I think we will need to keep the existing arg around as deprecated for 3 months.

Can you do something like this:
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/distributions/mvn_diag.py#L138

Perhaps default it to `keep_dims=None`, and then on the first line, `keepdims = keepdims if keep_dims is None else keep_dims`

You can move it to the last position right before name, and add back the docstring but note it's deprecated in the docstring. Date should be +3m from now.

Comment (brianwa84) [reply tensorflow_probability/python/stats/quantiles.py]: Also, this would only apply to public methods. Anything internal you can just change over.

**Comment by rushabh-v** [comment]
SHARD 0 and SHARD 1 are still failing. It says `build did not complete successfully`.

**Comment by brianwa84** [comment]
No worries, I can take a look at the failure when I merge. Comment above resolved now?

**Comment by rushabh-v** [comment]
I have added deprecation with date 2020-05-03, Should I add a test for the deprecation?

**Comment by brianwa84** [comment]
No test needed, thanks.

**Pull request merged**